src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -828,6 +828,23 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   <data name="ConfigCommandKeyNotFound" xml:space="preserve">
     <value>Key '{0}' not found.</value>
   </data>
+  <data name="Error_ConfigSetInvalidKey" xml:space="preserve">
+    <value>'{0}' is not a valid config key.</value>
```

Comment:
How about <value>'{0}' is not a valid key in config section.</value>?
Users might misunderstand that they could set keys in other sections in settings.